docs/cli/index.rst

```diff
@@ -0,0 +1,14 @@
+CLI Tools
+=========
+
+.. toctree::
+   :titlesonly:
+
+   baseplate-healthcheck: Is your service alive? <healthcheck>
```

Comment:
Worth making this a declarative sentence instead of interrogative? The other CLI tools describe what each thing does. This one poses a question that seems to infer that this tool checks for health. We could state as much and not leave that up to interpretation.

Reply:
SGTM. Fixed.